tests/test_openclhpp.cpp

```diff
@@ -1334,7 +1335,6 @@ static cl_mem clCreateBufferWithProperties_testBufferWithProperties(
 
 void testBufferWithProperties(void)
```

Comment:
Looks like this function is used somewhere not compatible with this `#if` guard change.